gammapy/conftest.py

```diff
@@ -12,6 +12,10 @@
 ## from the list of packages for which version numbers are displayed
 ## when running the tests ... this was added in Astropy 1.0
 try:
+    # Removing h5py for now to get rid of this travis-ci fail
+    # https://travis-ci.org/gammapy/gammapy/jobs/86565582#L1074
+    del PYTEST_HEADER_MODULES['h5py']
```

Comment:
I think they updated the h5py package to one that's Numpy-version independent, which seems like a bug, so I opened an issue a few hours ago: https://github.com/ContinuumIO/anaconda-issues/issues/486

I'm seeing this in other packages.

I think there's nothing we can do but wait until they fix it, unless we force the version to an older one.